docs/accounts/add.md

```diff
@@ -62,35 +60,37 @@ WAN or a private LAN via Wi-Fi. See below for more details if you have trouble.
 Available options are:
 
 * None: This doesn't use any transport security at all.
-* SSL/TLS (if available): SSL/TLS is used but the certificate isn't checked.
```

Comment:
What is the current method of connecting to a server with an invalid certificate? I feel this is the place to document it. (Or if it's not supported, to make a note of that)

Reply:
A dialog is displayed asking the user to manually accept the password. 

You can see this for yourself by using a hostname that isn't contained in the server's certificate, but points to your mail server. E.g. point `test.yourdomain.example` to one of the IPs of `imap.gmail.com`.

Reply:
OK I added a section for this case later on.